src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -1105,16 +1105,24 @@ InstanceKlass* SystemDictionary::load_shared_class(InstanceKlass* ik,
       if (fs.is_null_free_inline_type()) {
         // Pre-load inline class
         TempNewSymbol name = Signature::strip_envelope(sig);
+        log_info(class, preload)("Preloading class %s during loading of shared class %s. Cause: field type in LoadableDescriptors attribute", name->as_C_string(), ik->name()->as_C_string());
```

Comment:
@matias9927 Sorry I'm late here but shouldn't this read `Cause: a null-free non-static field is declared with this type` similar to the logging done in `ClassFileParser::post_process_parsed_stream`? Only below code actually checks for `is_class_in_loadable_descriptors_attribute`.